### ISSUE TITLE
External Event Drag and Drop #142

### ISSUE THREAD

**Issue by ARCHERS** [stefanuebe/vaadin-fullcalendar#198]
There was previous extensive discussion in issue #142 about improving external drag and drop functionality, with a small dev branch implementation in 4.2 (https://github.com/stefanuebe/vaadin_fullcalendar/tree/v.4.2.0_dev). However this does not appear to been included in fc version 6. 

Currently using Vaadin 24.6.0 and Fullcalendar/scheduler  6.2.2.

Are there plans to develop that?

**Comment by MarukoRook** [comment]
I agree with @ARCHERS  request. I would need to do a drag & drop starting from an external element. I tried to follow the steps in post #142 but I haven't been able to get it to work yet. Is it still valid or is it obsolete with version 6.x.x?

Thanks

**Comment by stefanuebe** [comment]
I'll try to have a look into it in the near future.

**Comment by stefanuebe** [comment]
Took some time, but it should be doable now. Check the demo in demo/src/main/java/org/vaadin/stefan/ui/view/demos/ExternalDragDemo.java
